domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapperTest.java

```diff
@@ -883,11 +885,18 @@ private FieldType mapAssertCommonFieldsAndReturn(FieldTypeEntity fieldTypeEntity
             fieldTypeEntity.setRegularExpression("SomeRegex");
 
             FieldTypeListItemEntity fieldTypeListItemEntity1 = new FieldTypeListItemEntity();
+            fieldTypeListItemEntity1.setLabel("label1");
```

Comment:
do we need to set the label on the fieldTypeListItemEntity items? Or only on the fixedListItems would be ok?

Reply:
Yes, unfortunately we need to, please.